content/applications/websites/website/optimize/google_analytics.rst

```diff
@@ -11,8 +11,8 @@ How to track your website's traffic in Google Analytics
   <https://support.google.com/analytics/answer/11583832>`_ are most likely still
   using **Universal Analytics**.
 
-  Odoo 15.0 is already adapted and ready for **Analytics 4**. After July 2023,
-  if you want to use Google Analytics tracking in versions before 15.0, you will
+  Odoo {LATEST_RELEASE} is already adapted and ready for **Analytics 4**. After July 2023,
+  if you want to use Google Analytics tracking in versions before {LATEST_RELEASE}, you will
```

Comment:
meaning change here

Reply:
That's intended. This page was updated in 14.0 to suggest upgrading to Odoo 15, but we should always suggest upgrading to whichever version was last released IMHO.

Reply:
I agree that we should push to upgrade to the latest release, but line 15 gives instructions that are version-specific. I would rephrase it.
"in older versions of Odoo" or something like that.